spot-vaults/contracts/_strategies/SpotAppraiser.sol

```diff
@@ -49,29 +46,34 @@ contract SpotAppraiser is Ownable, IBillBrokerPricingStrategy {
 
     uint256 private constant DECIMALS = 18;
     uint256 private constant ONE = (10 ** DECIMALS);
+    uint256 private constant SPOT_DR_DECIMALS = 8;
+    uint256 private constant SPOT_DR_ONE = (10 ** SPOT_DR_DECIMALS);
     uint256 public constant CL_ORACLE_DECIMALS = 8;
-    uint256 public constant CL_ORACLE_STALENESS_TRESHOLD_SEC = 3600 * 24; // 1 day
+    uint256 public constant CL_ORACLE_STALENESS_THRESHOLD_SEC = 3600 * 48; // 2 days
     uint256 public constant USD_LOWER_BOUND = (99 * ONE) / 100; // 0.99$
-    uint256 public constant AMPL_DUST_AMT = 1000 * (10 ** 9); // 1000 AMPL
-
-    /// @notice Address of the AMPL ERC-20 token contract.
-    IAMPL public immutable AMPL;
+    uint256 public constant AMPL_DUST_AMT = 25000 * (10 ** 9); // 25000 AMPL
```

Comment:
Increased this to 25,000 AMPL (above the discussed 10k), to match with the rollover vault's min-deployment-amount. Which means if someone griefs by sending 25k ampl into perp, the vault should immediately roll it over.